### PR TITLE
Add more presets to date filters

### PR DESCRIPTION
**Pull request by Hegazy360**
screenshot-localhost_4200-2019 09 19-16_03_57

### CODE REVIEW

File: addon/components/hyper-table/filters-renderers/date.js [hunk 1]
```diff
@@ -26,7 +26,11 @@ export default Component.extend({
 
   movingDateOptions: {
     'Today': 'today',
-    'Yesterday': 'yesterday'
+    'Yesterday': 'yesterday',
+    'This Week': 'This Week',
+    'Last Week': 'Last Week',
+    'This Month': 'This Month',
+    'This Year': 'This Year',
```
Comment (phndiaye) [inline addon/components/hyper-table/filters-renderers/date.js]: can you put the values in lowercase + underscore as they are used as "ids" and don't have to be humanized

File: app/templates/components/hyper-table.hbs [hunk 2]
```diff
@@ -119,11 +119,8 @@
 
   {{#if (eq _collection.length 0)}}
     <div class="hypertable__state hypertable__state--empty">
-      {{#if (has-block "inverse")}}
         {{yield to="inverse"}}
-      {{else}}
-        No Data
-      {{/if}}
```
Comment (phndiaye) [inline app/templates/components/hyper-table.hbs]: why this change ? 🤔

Comment (Hegazy360) [reply app/templates/components/hyper-table.hbs]: My bad shouldn't have committed that, mock data doesn't show otherwise

**Comment by phndiaye** [comment]
LGTM 👍
